lib/internal/perf/observe.js

```diff
@@ -6,7 +6,6 @@ const {
   ArrayPrototypeFilter,
   ArrayPrototypeIncludes,
   ArrayPrototypePush,
-  ArrayPrototypePushApply,
```

Comment:
Unrelated to this PR: Can we eventually add an eslint rule to avoid using this?

Reply:
What would be the rational for such a rule?

Reply:
V8 has a hard limit on function argument count and without explicitly checking the number, it is easy to make a mistake?

Reply:
Depending on use it can also be significantly faster. In undici we batch the .apply calls in our data url parser for performance: https://github.com/nodejs/undici/blob/89a46dd54f7d7db7513c435fac8042769ee9e9b5/lib/web/fetch/data-url.js#L657-L674